tools/buildsteps/defaultenv

```diff
@@ -59,7 +59,7 @@ case $XBMC_PLATFORM_DIR in
     DEFAULT_NDK_VERSION="21e" # NDK package version (newer API can be inside)
     DEFAULT_NDK_API="21" # Lollipop API level (21) defined in package ./sysroot/usr/include/android/api-level.h
     DEFAULT_XBMC_DEPENDS_ROOT=$WORKSPACE/tools/depends/xbmc-depends
-    DEFAULT_CONFIGURATION="Debug"
+    DEFAULT_CONFIGURATION="RelWithDebInfo"
```

Comment:
Should this be the default for all platforms?

Reply:
for me no problem set it for all them
but i am not the best person to ask
@fuzzard ?

Reply:
I dont really have an opinion either way.

Id probably just go as is for now until we find a need on other platforms. Most other platforms are more performant than android systems, so we may not really see any difference either way.